### PR TITLE
doc: Remove Getting Started Assistant documentation

### PR DESCRIPTION
**Pull request by divipillai**
Remove Getting Started Assistant from nRF Connect SDK
documentation as Getting Started Assistant tool is deprecated.
NCSDK-13800

Signed-off-by: divya pillai <divya.pillai@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_assistant.rst [hunk 1]
```diff
@@ -9,11 +9,8 @@ Installing automatically
 
 `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different apps that simplify installing the |NCS|.
 
-.. note::
-   When installing the |NCS| through the |GSA| app, you will not be able to build on the command line using west.
-
 On Windows or macOS, use the :ref:`gs_app_tcm`.
-On Linux, use the :ref:`gs_app_gsa` or follow the steps found on the :ref:`gs_installing` page.
+On Linux, use the `Installing using Visual Studio Code`_ or follow the steps found on the :ref:`gs_installing` page.
```
Comment (divipillai) [inline doc/nrf/gs_assistant.rst:13]: Do we need to mention about Visual Studio Code here as the user will be redirected to same gs_installing page from the Visual Studio Code. Thoughts?

File: doc/nrf/gs_assistant.rst [hunk 2]
```diff
@@ -9,11 +9,8 @@ Installing automatically
 
 `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different apps that simplify installing the |NCS|.
 
-.. note::
-   When installing the |NCS| through the |GSA| app, you will not be able to build on the command line using west.
-
 On Windows or macOS, use the :ref:`gs_app_tcm`.
-On Linux, use the :ref:`gs_app_gsa` or follow the steps found on the :ref:`gs_installing` page.
+On Linux, use the `Installing using Visual Studio Code <Installing on Linux_>`_ or follow the steps found on the :ref:`gs_installing` page.
```
Comment (Avalei) [inline doc/nrf/gs_assistant.rst:13]: Having "the" before the link currently sounds a little strange. I would suggest the following:

```suggestion
On Linux, use the `Installing using Visual Studio Code <Installing on Linux_>`_ instructions or follow the steps found on the :ref:`gs_installing` page.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -531,3 +531,8 @@ In addition to documentation related to the changes listed above, the following
   * :ref:`nrfxlib:nrf_modem` - Removed overview diagram.
   * ``AT socket`` page removed.
   * ``Extensions`` page removed.
+
+* Removed:
+
+  * Documentation on the Getting Started Assistant, as this tool is no longer in use.
+    Linux users can use the `Installing using Visual Studio Code <Installing on Linux_>`_ or follow the steps on the :ref:`gs_installing` page for |NCS| installation.
```
Comment (Avalei) [inline doc/nrf/releases/release-notes-changelog.rst:538]: Same as the previous note.

```suggestion
    Linux users can use the `Installing using Visual Studio Code <Installing on Linux_>`_ instructions or follow the steps on the :ref:`gs_installing` page for |NCS| installation.
```

**Review by Avalei — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -169,6 +169,9 @@ The change prompted some changes in the CMake for the module, notably:
 Documentation
 =============
 
-|no_changes_yet_note|
+* Removed:
+
+  * Documentation on the Getting Started Assistant, as this tool is no longer in use.
+    Linux users can use the `Installing using Visual Studio Code <Installing on Linux_>`_ instructions or follow the steps on the :ref:`gs_installing` page for |NCS| installation.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:175]:
```suggestion
    Linux users can install the |NCS| by using the `Installing using Visual Studio Code <Installing on Linux_>`_ instructions or by following the steps on the :ref:`gs_installing` page.
```

**Review by greg-fer — COMMENTED** [review]
Just one comment.

**Review by greg-fer — APPROVED** [review]

**Pull request merged**